content/docs/manuals/extending-oakestra/install_addon.md

```diff
@@ -0,0 +1,47 @@
+---
+title: "Install an Addon"
+summary: ""
+draft: false
+weight: 2
```

Comment:
Maybe "install addons" should come before "create addons" as most people would only want to use existing addons? Also this documentation is written with basic Addon description that should be read first. 

If yes, please swap the weight of create and install docs